app/services/api/v1/users/ecf_users_query.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Api::V1::Users
+  class ECFUsersQuery
```

Comment:
maybe instead of nesting it as such `api/v1/users/ecf_users_query.rb` we could nest this query as `api/v1/ecf/users_query.rb`. Let me know your thoughts

Reply:
Good call, done.